src/frontend/src/lib/components/ui/ButtonMenu.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+	export let label: string;
+	export let onClick: () => void;
```

Comment:
Not sure what we defined last time we discussed this. Aren't sticking to the current pattern for consistency - bubbling event- for now?

```
// ButtonMenu
<button on:click />

// Consumer
<ButtonMenu on:click={myFunction} />
```

Reply:
you mean, to reduce the number of props

actually, agreed, better

Reply:
> component would have the event on:click, right?

Only if within the component `on:click` is present

Reply:
It's Svelte v5 this pattern disappear but, in our code base, it is use everywhere. So for consistency I would stick to that but, not sure anymore what we decided when we discussed that.

Reply:
perfect, will do!